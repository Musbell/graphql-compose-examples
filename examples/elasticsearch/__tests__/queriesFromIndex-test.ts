import { graphql } from 'graphql';
import meta from '../index';

describe('elasticsearch > queries', () => {
  meta.queries.forEach(({ query, title }) => {
    it(title, async () => {
      const result = await graphql({
        schema: meta.schema,
        source: query,
        contextValue: {},
      });
      expect(result).toMatchSnapshot();
    });
  });
});
