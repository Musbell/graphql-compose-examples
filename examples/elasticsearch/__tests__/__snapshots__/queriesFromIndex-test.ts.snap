// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`elasticsearch > queries Elastic search API 5.6 1`] = `
Object {
  "data": Object {
    "elastic56": null,
  },
  "errors": Array [
    [GraphQLError: âœ‹ ðŸ›‘ I don't have public elasticsearch instance for demo purposes. 
ðŸš€ Demo will work if you provide public elasticsearch instance url 
ðŸš€ in query argument \`host: "http://user:pass@example.com:9200"\` 
],
  ],
}
`;

exports[`elasticsearch > queries Elastic search API 6.8 1`] = `
Object {
  "data": Object {
    "elastic68": null,
  },
  "errors": Array [
    [GraphQLError: âœ‹ ðŸ›‘ I don't have public elasticsearch instance for demo purposes. 
ðŸš€ Demo will work if you provide public elasticsearch instance url 
ðŸš€ in query argument \`host: "http://user:pass@example.com:9200"\` 
],
  ],
}
`;

exports[`elasticsearch > queries Elastic search API 7.7 1`] = `
Object {
  "data": Object {
    "elastic77": null,
  },
  "errors": Array [
    [GraphQLError: âœ‹ ðŸ›‘ I don't have public elasticsearch instance for demo purposes. 
ðŸš€ Demo will work if you provide public elasticsearch instance url 
ðŸš€ in query argument \`host: "http://user:pass@example.com:9200"\` 
],
  ],
}
`;
